pkg/kf/internal/utils/cli/async_flags.go

```diff
@@ -103,3 +103,40 @@ func (flags *AsyncFlags) WaitFor(
 		}
 	})
 }
+
+type AsyncIfStoppedFlags struct {
+	AsyncFlags
+	async_if_stopped bool
+}
+
+// Add adds the async and async_if_stopped flags to the cobra command
+func (flags *AsyncIfStoppedFlags) Add(cmd *cobra.Command) {
+	flags.AsyncFlags.Add(cmd)
+
+	cmd.Flags().BoolVarP(
+		&flags.async_if_stopped,
+		"async-if-stopped",
```

Comment:
Could this be "always-wait-for-ready" with the default of no flag being false? That also gives us a little more leeway to use this on other non-app based commands if it's needed in the future.

The description would be something like "Force the CLI to wait even if the mutation doesn't impact the resource."

Reply:
Hmm, how should that flag interact when `--async` is provided as a CLI option? i.e. should one override the other?

Reply:
I think that's reasonable. With that behavior a flag named like `--no-short-circuit-wait`  might go better?

Reply:
sgtm, done.